hw25_custom_map/js/custom_map.js

```diff
@@ -0,0 +1,16 @@
+function customMap(callback){
+    let arr = [];
+    for (let i=0; i<this.length; i++)  {
+        arr.push(callback(this[i]))
```

Comment:
The default `map` has more than one argument ) Please supplement the line with `;`

Reply:
supplement the line with arguments